bin/BuildSystem/BuildSystemBase.py

```diff
@@ -240,18 +240,17 @@ def internalPostInstall(self):
 
             regexp = re.compile('{.*} (.*)')
             for f in files:
-                expectedPdb = os.path.splitext(f)[0] + '.pdb'
-                if not os.path.exists(expectedPdb):
+                if not os.path.exists(os.path.splitext(f)[0] + '.pdb'):
                     peparserOutput = CraftCore.cache.getCommandOutput(f"peparser", f"--pdb {f}")[1].strip()
                     pdbs = regexp.findall(peparserOutput)
+
+                    if not os.path.basename(f) == "icudt58.dll":
+                        assert len(pdbs) > 0, f'No pdb file available: {f}'
```

Comment:
I'd prefer a warning and no special treatment for icu

Reply:
I hate the special treatment for icu as well. I'd argue that if the pdb is missing it's an issue in the build system that should be fixed. And if there's only a warning we most certainly will miss it. If we find out that there are a lot of these issues and it's unreasonable to fix them all at this point, we could switch to a warning?

Reply:
then at least use a regexp to match :P 

I also expect the next full rebuild to be painful :D

Reply:
Hehehehe... likely :-\ 
But we can make it a list of exceptions then and take care of them one by one... 
Or we make it a warning and grep for it in the logs.. are we going to have a full log of the full rebuild?

Reply:
Made it a regexp

Reply:
We keep the full logs of the last n builds.